activity.py

```diff
@@ -70,9 +70,9 @@ def __init__(self, handle):
         vt.show()
 
         if platform.machine().startswith('arm'):
-            vt.feed_child("echo \"RiverHex, this activity won't run on this computer\"; sleep 10; exit\n", -1)
+            vt.feed_child("echo \"RiverHex, this activity won't run on this computer\"; sleep 10; exit\n")
         else:
-            vt.feed_child("cd $SUGAR_BUNDLE_PATH/bin; ./bin/riverhexactivity-activity; exit\n", -1)
+            vt.feed_child("cd $SUGAR_BUNDLE_PATH/bin; ./bin/riverhexactivity-activity; exit\n")
```

Comment:
Remove the `exit` commands. It force quits the actvity. Also, change `./bin/riverhexactivity-activity` to `./riverhexactivity-activity` (args required)